cookbooks/hangops-jobbot/recipes/nodejs.rb

```diff
@@ -51,3 +51,9 @@
           '/srv/hangops-jobbot/vendor/' \
           'hubot-slack-whitelist-middleware --save'
 end
+
+# Install hubot-rate-limit
+execute 'install-rate-limit' do
+  command 'cd /srv/hangops-jobbot;' \
+  'npm install hubot-rate-limit --save'
+end
```

Comment:
What's this Chef recipe doing then? Everything in `package.json` should be pulled in as part of the build, when you run `npm install`.